src/pages/fundamentals/pages/appendix/pages/config-annotations/pages/view/pages/chart.adoc

```diff
@@ -7,7 +7,18 @@ Chart renders a visually appealing graph/chart based on an open source HTML5 bas
 <<view-config-annotation-section>>
 
 .Allowed Children Components
-TODO
+* Line
```

Comment:
"Children Components" has been used previously in documentation refers to other components that are rendered under the given component via a nested type.

Usually this would read:
```asciidoc
None. `@Chart` should decorate a field having a collection with type `DataGroup` (e.g. `List<DataGroup>`.)
```

Consider moving the chart types to a new section.